doc/manipulating_images/manipulating_images.rst

```diff
@@ -1,236 +1,143 @@
-    following directories (in order of priority, if present):
-
-     * the folder specified by `data_dir` parameter in the fetching function
-     * the global environment variable `NILEARN_SHARED_DATA`
-     * the user environment variable `NILEARN_DATA`
-     * the `nilearn_data` folder in the user home folder
-
-    The two different environment variables (NILEARN_SHARED_DATA and
-    NILEARN_DATA) are provided for multi-user systems, to distinguish a
-    global dataset repository that may be read-only at the user-level.
-    Note that you can copy that folder to another user's computers to
-    avoid the initial dataset download on the first fetching call.
-
-
-Loading your own data: filenames or Nibabel objects
-----------------------------------------------------
-
-Function in nilearn can take filenames or nibabel in-memory object.
-
-Using your own data images in nilearn is as simple as giving the file
-name, or list of file name strings ::
-
-    >>> # dataset folder contains subject1.nii and subject2.nii
-    >>> from nilearn.image import smooth_img
-    >>> result_img = smooth_img(['dataset/subject1.nii', 'dataset/subject2.nii']) # doctest: +SKIP
-
-``result_img`` is a 4D in-memory image, containing the data of both
-subjects.
-
-**Filename matching** Nilearn also provides a "wildcard" pattern to list
-many files with one expression:
-
- * **Matching multiple files**: suppose the dataset folder contains
-   subject_01.nii to subject_03.nii ``dataset/subject_*.nii`` is a glob
-   expression matching all filenames::
-
-    >>> # Example with a smoothing process:
-    >>> from nilearn.image import smooth_img
-    >>> result_img = smooth_img("dataset/subject_*.nii") # doctest: +SKIP
-
- * **Expanding the home directory** ``~`` is expanded to your home
-   directory::
-
-    >>> result_img = smooth_img("~/dataset/subject_01.nii") # doctest: +SKIP
-
-.. topic:: **Python globbing**
+Nilearn comes with many simple functions for simple data preparation and
+transformation. Note that if you want to perform these operations while
+loading the data into a data matrix, most are also integrated in the
+:ref:`masker objects <masker_objects>`.
 
-    For more complicated use cases, Python also provides functions to work
-    with file paths, in particular, :func:`glob.glob`.
+.. currentmodule:: nilearn
 
-    .. warning::
 
-        Unlike nilearn's path expansion, the result of :func:`glob.glob` is
-        not sorted and depending on the computer you are running they
-        might not be in alphabetic order. We advise you to rely on
-        nilearn's path expansion.
+* Computing the mean of images (along the time/4th dimension):
+  :func:`nilearn.image.mean_img`
+* Applying numpy functions on an image or a list of images:
+  :func:`nilearn.image.math_img`
+* Swapping voxels of both hemisphere (e.g., useful to homogenize masks
+  inter-hemispherically):
+  :func:`nilearn.image.swap_img_hemispheres`
```

Comment:
Perhaps it would be nice to show how to homogenize the masks using swap_img and math_img?